--- /dev/null	2019-10-23 20:19:32.602763995 -0700
+++ b/puppetlabs-apache.gemspec	2019-10-28 12:43:41.741170991 -0700
@@ -0,0 +1,12 @@
+Gem::Specification.new do |s|
+  s.name        = 'puppetlabs-apache'
+  s.version     = '1.10.0'
+  s.date        = '2017-05-17'
+  s.summary     = "Installs, configures, and manages Apache virtual hosts, web services, and modules."
+  s.description = s.summary
+  s.authors     = ["Puppet Labs"]
+  s.email       = ''
+  s.files       = %w(LICENSE Rakefile) + Dir.glob('{lib,spec}/**/*')
+  s.homepage    = 'https://github.com/puppetlabs/puppetlabs-apache'
+  s.license     = 'Apache 2.0'
+ end
