--- /dev/null	2019-10-23 20:19:32.602763995 -0700
+++ b/puppet-oslo.gemspec	2019-10-28 08:59:29.609410185 -0700
@@ -0,0 +1,12 @@
+Gem::Specification.new do |s|
+  s.name        = 'puppet-oslo'
+  s.version     = '11.3.0'
+  s.date        = '2016-08-21'
+  s.summary     = "Puppet oslo module"
+  s.description = s.summary
+  s.authors     = ["OpenStack Contributors"]
+  s.email       = ''
+  s.files       = %w(LICENSE README.md Rakefile) + Dir.glob('{lib,spec}/**/*')
+  s.homepage    = 'https://github.com/openstack/puppet-oslo.git'
+  s.license     = 'Apache 2.0'
+ end
