From 90e50eedd7c8ba0d1e1c17ba678525630bb7e023 Mon Sep 17 00:00:00 2001
From: Bryan Jen <bryan.jen@gmail.com>
Date: Fri, 16 Dec 2016 15:57:46 +0000
Subject: [PATCH 2/2] (maint) Fix the vhost ssl template correctly this time

---
 templates/vhost/_ssl.erb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/templates/vhost/_ssl.erb b/templates/vhost/_ssl.erb
index c92f70c8..e70efebd 100644
--- a/templates/vhost/_ssl.erb
+++ b/templates/vhost/_ssl.erb
@@ -49,7 +49,7 @@
   <%- if @ssl_stapling_timeout && scope.function_versioncmp([@apache_version, '2.4']) >= 0 -%>
   SSLStaplingResponderTimeout <%= @ssl_stapling_timeout %>
   <%- end -%>
-  <%- if (not @ssl_stapling_return_errors.nil? && scope.function_versioncmp([@apache_version, '2.4']) >= 0) -%>
+  <%- if (not @ssl_stapling_return_errors.nil?) && (scope.function_versioncmp([@apache_version, '2.4']) >= 0) -%>
   SSLStaplingReturnResponderErrors <%= scope.function_bool2httpd([@ssl_stapling_return_errors]) %>
   <%- end -%>
 <% end -%>
-- 
2.17.1

