--- /dev/null	2019-10-23 20:19:32.602763995 -0700
+++ b/puppet-etcd.gemspec	2019-10-28 10:02:42.678631063 -0700
@@ -0,0 +1,12 @@
+Gem::Specification.new do |s|
+  s.name        = 'puppet-etcd'
+  s.version     = '1.11.0'
+  s.date        = '2016-08-21'
+  s.summary     = "Installs and configures etcd"
+  s.description = s.summary
+  s.authors     = ["Cristian Falcas"]
+  s.email       = ''
+  s.files       = %w(LICENSE README.md Rakefile) + Dir.glob('{lib,spec}/**/*')
+  s.homepage    = 'https://github.com/cristifalcas/puppet-etcd'
+  s.license     = 'Apache 2.0'
+ end
