--- /dev/null	2019-10-23 20:19:32.602763995 -0700
+++ b/puppet-openstacklib.gemspec	2019-10-28 19:41:18.343027332 -0700
@@ -0,0 +1,12 @@
+Gem::Specification.new do |s|
+  s.name        = 'puppet-openstacklib'
+  s.version     = '11.3.0'
+  s.date        = '2017-08-21'
+  s.summary     = "Puppet OpenStack Libraries"
+  s.description = s.summary
+  s.authors     = ["Puppet Labs and OpenStack Contributors"]
+  s.email       = ''
+  s.files       = %w(LICENSE README.md Rakefile) + Dir.glob('{lib,spec}/**/*')
+  s.homepage    = 'https://launchpad.net/puppet-openstacklib'
+  s.license     = 'Apache 2.0'
+ end
