--- /dev/null	2019-10-23 20:19:32.602763995 -0700
+++ b/puppet-certmonger.gemspec	2019-10-28 07:51:11.303843437 -0700
@@ -0,0 +1,12 @@
+Gem::Specification.new do |s|
+  s.name        = 'puppet-certmonger'
+  s.version     = '1.1.1'
+  s.date        = '2016-10-10'
+  s.summary     = "Puppet certmonger module"
+  s.description = s.summary
+  s.authors     = ["Puppet saz "]
+  s.email       = ''
+  s.files       = %w(LICENSE README.md Rakefile) + Dir.glob('{lib,spec}/**/*')
+  s.homepage    = 'https://github.com/earsdown/puppet-certmonger'
+  s.license     = 'Apache 2.0'
+ end
