diff --git a/mtce-guest/src/Makefile b/mtce-guest/src/Makefile
index 40dd933..ba6e029 100644
--- a/mtce-guest/src/Makefile
+++ b/mtce-guest/src/Makefile
@@ -31,8 +31,8 @@ LDLIBS = $(EXTRALDFLAGS) -lstdc++ -ldaemon -lcommon -lfmcommon -ljson-c -levent
 INCLUDES = -I. -I/usr/include/mtce-common -I/usr/include/mtce-daemon
 
 build: $(OBJS)
-	$(CXX) $(CCPFLAGS) $(AGENT_OBJS)  $(LDLIBS) -L. -o guestAgent
-	$(CXX) $(CCPFLAGS) $(SERVER_OBJS) $(LDLIBS) -L. -o guestServer
+	$(CXX) $(CCPFLAGS) $(LDFLAGS) $(AGENT_OBJS)  $(LDLIBS) -L. -o guestAgent
+	$(CXX) $(CCPFLAGS) $(LDFLAGS) $(SERVER_OBJS) $(LDLIBS) -L. -o guestServer
 
 .cpp.o:
 	$(CXX) $(INCLUDES) $(CCPFLAGS) $(EXTRACCFLAGS) -c $< -o $@
