--- /dev/null	2019-10-28 19:50:04.372284581 -0700
+++ b/puppetlabs-postgresql.gemspec	2019-10-29 18:57:08.102259309 -0700
@@ -0,0 +1,12 @@
+Gem::Specification.new do |s|
+  s.name        = 'puppetlabs-postgresql'
+  s.version     = '4.8.0'
+  s.date        = '2016-12-22'
+  s.summary     = "Offers support for basic management of PostgreSQL databases."
+  s.description = s.summary
+  s.authors     = ["Inkling/Puppet Labs"]
+  s.email       = ''
+  s.files       = %w(LICENSE README.md Rakefile) + Dir.glob('{lib,spec}/**/*')
+  s.homepage    = 'https://github.com/puppetlabs/puppetlabs-postgresql'
+  s.license     = 'Apache 2.0'
+ end
